recap/examples/log.rs

```diff
@@ -10,6 +10,7 @@ use std::error::Error;
     \s+
     (?P<baz>\S+)
   "#)]
+#[allow(dead_code)]
```

Comment:
the fields were printed with debug formatted but rustc complained they were unused so I'm adding `#[allow(dead_code)]` to allow for a cleaner compile